### ISSUE TITLE
different resources sharing same repository

### ISSUE THREAD

**Issue by nitedog** [w3c/wai-resources#16]
See "planning", "improving", and "org-policy" - they use the same repository but are actually separate resources :( - tool did not account for such situations

**Comment by yatil** [comment]
Yes, you just have to point to the same repository from all the resources, that is what I do with the tutorials, too.

**Comment by nitedog** [comment]
yes but i think the script adds slashes "/" which breaks things

**Comment by yatil** [comment]
Where? I can’t follow you :-)

**Comment by yatil** [comment]
Oh, you can’t deep link into a repository. You can just specify a repository.

**Comment by yatil** [comment]
> "wai-planning-and-implementation/improving"

needs to be 

> "wai-planning-and-implementation"

**Comment by nitedog** [comment]
OK. It would be good to be able to optionally override the generated link for "Draft", though. Currently the tool generates "https://w3c.github.io/wai-planning-and-implementation/" which is wrong (and broken). In some cases, like for the dynamic planning guide ("managing") the filename "Overview.html" needs to be added for the link to work - not sure why, though.

**Comment by yatil** [comment]
Overview.html needs to be added because it is not a recognized file index and therefore is not picked up. If you rename it to index.html it works.

However adding a possibility to specify an arbitrary Draft location is a good thing. _adds to the list_
